static/flow/views/fullscreen-widget.js

```diff
@@ -0,0 +1,58 @@
+//
+// A widget to make app fullscreen
+//
+var FullScreenWidget = function() {
+
+    var fullscreenContainer = jQuery('#fullscreen-widget');
+    var fullscreenButton  = jQuery('<div>', {id:'fullscreen-button'} );
+    fullscreenButton.appendTo(fullscreenContainer);
+    var fullscreen = false;
+    fullscreenButton.click(function(e) {
+        if (screenfull.enabled) {
+            screenfull.toggle();
+        }
+    });
+    //
+    // function to enter or exit fullscreen
+    //
+    function requestFullScreen(element) {
+        // Supports most browsers and their versions.
+        var requestMethod = element.requestFullScreen || element.webkitRequestFullScreen || element.mozRequestFullScreen || element.msRequestFullScreen;
+
+        if (requestMethod) { // Native full screen.
+            requestMethod.call(element);
+        } else if (typeof window.ActiveXObject !== "undefined") { // Older IE.
+            var wscript = new ActiveXObject("WScript.Shell");
+            if (wscript !== null) {
+                wscript.SendKeys("{F11}");
+            }
+        }
+
+        fullscreen = !fullscreen;
+        if(fullscreen){
+            $('#fullscreen-button').addClass("exit");
+        }
+        else{
+            $('#fullscreen-button').removeClass("exit");
+        }
+    }
+
+    /*!
+    * screenfull
+    * v3.3.2 - 2017-10-27
+    * (c) Sindre Sorhus; MIT License
+    */
```

Comment:
This seems like a slightly sketchy way to include a library, is this standard across the project?

Reply:
That's what I thought!  This is straight from what from the SEPA project does.  I was told to reuse existing CC implementations for common functionality.  Hmmm....